input/fsh/au-erequesting-diagnosticrequest.fsh

```diff
@@ -1,7 +1,7 @@
-Profile: AUeRequestingServiceRequest
+Profile: AUeRequestingDiagnosticRequest
 Parent: AUBaseDiagnosticRequest
-Id: au-erequesting-servicerequest
-Title: "AU eRequesting ServiceRequest"
+Id: au-erequesting-diagnosticrequest
```

Comment:
Commenting here to get an inline comment. 

Please update AU Variance Statement (link and title).

Reply:
Amended, thanks.